ui/components/app/transaction-status-label/transaction-status-label.js

```diff
@@ -96,20 +97,29 @@ export default function TransactionStatusLabel({
     }
   }
   ///: END:ONLY_INCLUDE_IF
-
-  return (
+  return shouldShowTooltip ? (
```

Comment:
Keep the tooltip for same component here
<img width="1279" alt="Screenshot 2024-12-18 at 18 24 09" src="https://github.com/user-attachments/assets/54dc4a45-a819-4df9-aa2d-ca68b0885a45" />